### DISCUSSION TITLE
Unable to import Google provider

### DISCUSSION THREAD

**Discussion by rsotogar** [apache/airflow#19391] [Q&A]
I've added the Google dependency in the requirements.txt file and successfully installed it, but the DAG fails to import. Any idea why? I am using MWAA in AWS with Airflow version 2.0.2.

<img width="412" alt="Screenshot 2021-11-03 at 17 48 44" src="https://user-images.githubusercontent.com/59261157/140105632-7e9d7d64-e1ad-4f14-9b5c-fed214c7f1ea.png">

<img width="635" alt="Screenshot 2021-11-03 at 17 49 07" src="https://user-images.githubusercontent.com/59261157/140105674-444e528d-5cdd-470e-a0bf-b12b44c27aed.png">

It may be because I need to update the environment every time I upload the requirements.txt file to S3? I have a Github actions that delivers it to the required path, but I didnt try to update the environment manually.

Thanks!

**Comment by potiuk** [comment]
This is the likely reason (https://pypi.org/project/apache-airflow-providers-google/):

PIP package | Version required
-- | --
apache-airflow | >=2.1.0


MWAA does not have yet support for 2.1.0+

**Comment by rsotogar** [comment]
I removed the dependencies that may have caused this issue but the UI cannot still open. I will contact MWAA support
